### DISCUSSION TITLE
How to globally apply biome.json to multiple workplaces or projects?

### DISCUSSION THREAD

**Discussion by coolcode** [biomejs/biome-vscode#38] [Q&A] [answered]
As official docs mentioned, `biome.json` has to be put at the root of workplace. Is it possible to make one file `biome.json` or `biome-global.json` as a default configuration and apply to multiple workplaces or projects?

**Comment by nhedger** [comment, marked as answer]
Unfortunately, Biome does not yet handle multiple workspaces.

If all your projects happen to be a part of the same monorepo, I'd suggest putting the `biome.json` at the root of your monorepo.

**Reply by ghandic** [reply]
As an alternative you can symlink so its available in both

**Comment by j-murata** [comment]
Hi @nhedger, I am not sure if this discussion is the right place to post this, but I would like to comment.

When developing in a large monorepo with VSCode, it is often the case that each module has different Lint rules to apply.
I recently tried to [use Biome in monorepo](https://biomejs.dev/guides/big-projects/) and was quite confused because the `biome.json` configurations in the current working directory (or nearest parent directory) were not reflected on the UI feedback.

Perhaps [the `eslint.workingDirectories` option of `vscode-eslint`](https://github.com/microsoft/vscode-eslint#:~:text=how%20the%20working%20directories%20ESLint%20is%20using%20are%20computed) is a similar feature. If such an option is implemented in `biome-vscode`, more users may be willing to replace ESLint with Biome.

**Reply by ematipico** [reply]
That wouldn't work because our LSP still doesn't support workspaces.

Ideally, a user wouldn't need any option in VSCode, so users that use other editors can support it too.

Follow https://github.com/biomejs/biome/issues/1573

**Reply by dikaio** [reply]
@j-murata take a look at [ultracite](https://github.com/haydenbleasel/ultracite), I'm often using it for monorepos, specifically turborepos for nextjs projects.
